src/main/flight/mixer.c

```diff
@@ -910,7 +910,7 @@ uint16_t yawPidSumLimit = currentPidProfile->pidSumLimitYaw;
     motorMixRange = motorMixMax - motorMixMin;
 
     if (IS_RC_MODE_ACTIVE(BOXBLACKBOX)) {
-        applyAirMode(motorMix, motorMixMax);
+        applyAirMode(motorMix, motorMixMax, isAirmodeActive() ? 1.0f : 0.75f);
```

Comment:
for testing purposes at least I would love to see us being able to change the active airmode percent. I know a lot of tiny whoop pilots who don't fly airmode because of some issues it gives when you bump into things. So for at least testing purposes I love the idea of being able to mess with this. :)